src/components/app/MenuButtons/index.js

```diff
@@ -71,7 +71,7 @@ const MenuButtons = ({
         className="menuButtonsLink"
         title="Open the documentation in a new window"
       >
-        Docs…
+        Docs<img src="./res/img/svg/open-in-new.svg" />
```

Comment:
I think it would be much better if you add it in CSS using background-image and adding some padding to this element. The class `menuButtonsLink` is styled in https://github.com/firefox-devtools/profiler/blob/ccbfde60ab5ddfaea8bd65a01ffba896741c2092/src/components/app/MenuButtons/index.css#L24-L33.

Usually we use `<img>` only for images that are actual content. In this case the icon is pure decoration, and that's why it's better to define it in CSS.